mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.thefuntasty.mvvm.test
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import io.github.plastix.rxschedulerrule.RxSchedulerRule
+import org.junit.Rule
+
+/**
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/ef3136579489660a161140b43ae3c9f625f10e7b/mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt#L7"></span>An abstract class without an abstract member can be refactored to a concrete class.</td>
    </tr>
  </tbody>
</table>